loggly/loggly.go

```diff
@@ -27,6 +28,7 @@ func init() {
 
 	r := &router.Route{
 		Adapter: "loggly",
+		FilterName: os.Getenv(filterNameEnvVar),
```

Comment:
Looks like LogSpout will do the right thing in the case that FilterName is ""

https://github.com/gliderlabs/logspout/blob/95fdf26b9d3aaf5030f894ce6caf4f5abb6e660d/router/types.go#L99

Totally makes sense since Go will set those values to the default value for the type if the attribute is not initialized.